tdo.sh

```diff
@@ -151,7 +172,12 @@ new_note() {
     root="$NOTES_DIR"
     note_file="$root/notes/$1.md"
     template="$root/templates/note.md"
+    [ ! -f "$note_file" ] && new_file="true" || new_file="false"
     create_file "$note_file" "$template"
+    if [ "$new_file" = "true" ]; then
+      [ $filename_as_title = "true" ] && echo -e "# $1" >>"$note_file"
+      [ $add_new_note_timestamp = "true" ] && add_timestamp "$note_file" "$note_timestamp_format"
+    fi
```

Comment:
This can be improved, by removing an extra condition, and checks on boolean
```suggestion
    create_file "$note_file" "$template"
    if [ ! -f "$note_file" ]; then
      $filename_as_title && echo -e "# $1" >>"$note_file"
      $add_new_note_timestamp && add_timestamp "$note_file" "$note_timestamp_format"
    fi
```